services/core-web/src/components/mine/Summary/MineSummary.js

```diff
@@ -94,6 +94,7 @@ export class MineSummary extends Component {
         <br />
         {this.props.partyRelationships && this.props.partyRelationships.length > 0 && (
           <div>
+            <br />
             <Row>
```

Comment:
This fixes the issue where the link on the map (on the MineHeader) cannot be clicked because the row covers it, I moved the row down slightly so the link is now clickable.